docs/en/advanced_guides/add_dataset.md

```diff
@@ -1,5 +1,54 @@
 # Add New Datasets
 
+## Data configuration
+
+`train_dataloader`, `val_dataloader` and `test_dataloader` in config file are the variables for data configuration, to define the arguments that are used in datasets and dataloaders.
+
+Here is an example of dataloader configuration:
+
+```python
+train_dataloader = dict(
+    batch_size=4,
+    num_workers=4,
+    persistent_workers=True,
+    sampler=dict(type='InfiniteSampler', shuffle=True),
+    dataset=dict(
+        type=dataset_type,
+        data_root=data_root,
+        data_prefix=dict(
+            img_path='images/training', seg_map_path='annotations/training'),
+        pipeline=train_pipeline))
+val_dataloader = dict(
+    batch_size=1,
+    num_workers=4,
+    persistent_workers=True,
+    sampler=dict(type='DefaultSampler', shuffle=False),
+    dataset=dict(
+        type=dataset_type,
+        data_root=data_root,
+        data_prefix=dict(
+            img_path='images/validation',
+            seg_map_path='annotations/validation'),
+        pipeline=test_pipeline))
+test_dataloader = val_dataloader
+```
+
+- `train_dataloader`, `val_dataloader` and `test_dataloader`: The [`config`](https://github.com/open-mmlab/mmcv/blob/master/docs/en/understand_mmcv/config.md)s to build dataset instances for model training, validation and testing by
```

Comment:
The setting of dataloaders  works for model training, validation and testing that are by `build and registry` mechanism